test/acceptance/integration/ac_mdw/__init__.py

```diff
@@ -1 +1,24 @@
-__author__ = 'Jon'
+# -*- coding: utf-8 -*-
```

Comment:
Yes, PEP has a plugin for AC(PAP), where PeP checks the action in the AC-URL, and with this action interact with AC and KS